### PR TITLE
Add dependabot configuration

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: patch
See: https://www.flowdock.com/app/rulemotion/d-build-pipelines/threads/mqmgXw4ASi4gKIs1hQdfJkH0OTy

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+# The ignored dependencies are so that we get open PRs that result proper nested changelogs
+
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (sradevski) [inline .github/dependabot.yml:13]: Isn't daily too frequent? We might get too much noise if it is daily, weekly might be better. Also, is dependabot something we decided to use company-wide or just on the sdk?

Comment (thgreasi) [reply .github/dependabot.yml:13]: We are using it in JF and Lucian pointed me to their config, which I then used as basis for this. We have also extended VB to detect the change type from the title prefix, which this config instructs dependabot to add. So I was just thinking whether we could have this a try in the SDK. Still haven't decided though.

Comment (sradevski) [reply .github/dependabot.yml:13]: @thgreasi yeah we can give it a go I guess, but I still think daily might be too much :) I'll leave it up to you to decide, and feel free to self-approve.

**Comment by thgreasi** [comment]
@balena-ci I self-certify!

**Pull request merged**